apps/web/src/app/api/documentLogs/evaluation-results/route.ts

```diff
@@ -0,0 +1,30 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { fetchEvaluationResultsByDocumentLogs } from '@latitude-data/core/services/evaluationResults/fetchEvaluationResultsByDocumentLogs'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+
+export const GET = errorHandler(
+  authHandler(
+    async (
+      request: NextRequest,
+      {
+        workspace,
+      }: {
+        workspace: Workspace
+      },
+    ) => {
+      const documentLogIds = request.nextUrl.searchParams.get('ids')?.split(',')
+      if (!documentLogIds?.length) {
+        return NextResponse.json({}, { status: 200 })
```

Comment:
Better return an empty array. This way you don't need to defend on the frontend. Just iterate the list

Reply:
I return an empty object because the data structure it returns is an object where the keys are documentLogIds and the values are Lists of EvaluationResults. If I were to do this as a list it would be like a matrix, and that is unnecessarily complex to work with/find things.